tests/resources/defects.json

```diff
@@ -11247,17 +11247,6 @@
           "psv"
         ],
         "deficiencies": [
-          {
-            "ref": "46.1.b.ii",
```

Comment:
46.1.b.ii is gone, but where is 46.1.b.i?

Reply:
![image](https://user-images.githubusercontent.com/50110006/72746531-3fdabb80-3baa-11ea-9c7d-80bd89460aa8.png)

Reply:
Please look at line 11297